tests/integrations/subscription.spec.ts

```diff
@@ -48,244 +48,489 @@ async function runTest(page: Page) {
 	]);
 
 	await fieldset.name.pressSequentially('Edmund');
-	await expect(fieldset.logs).toHaveText([
-		'form.id: 1',
-		'form.errorId: 1',
-		'form.descriptionId: 1',
-		'form.initialValue: 1',
-		'form.value: 7',
-		'form.key: 1',
-		'form.dirty: 2',
-		'form.valid: 1',
-		'form.errors: 1',
-		'form.allErrors: 1',
-		'name.id: 1',
-		'name.formId: 1',
-		'name.errorId: 1',
-		'name.descriptionId: 1',
-		'name.initialValue: 1',
-		'name.value: 7',
-		'name.key: 1',
-		'name.dirty: 2',
-		'name.valid: 1',
-		'name.errors: 1',
-		'message.id: 1',
-		'message.formId: 1',
-		'message.errorId: 1',
-		'message.descriptionId: 1',
-		'message.initialValue: 1',
-		'message.value: 1',
-		'message.key: 1',
-		'message.dirty: 1',
-		'message.valid: 1',
-		'message.errors: 1',
-	]);
+	if (process.platform === 'darwin') {
+		await expect(fieldset.logs).toHaveText([
+			'form.id: 1',
+			'form.errorId: 1',
+			'form.descriptionId: 1',
+			'form.initialValue: 1',
+			'form.value: 4',
+			'form.key: 1',
+			'form.dirty: 1.5',
+			'form.valid: 1',
+			'form.errors: 1',
+			'form.allErrors: 1',
+			'name.id: 1',
+			'name.formId: 1',
+			'name.errorId: 1',
+			'name.descriptionId: 1',
+			'name.initialValue: 1',
+			'name.value: 4',
+			'name.key: 1',
+			'name.dirty: 1.5',
+			'name.valid: 1',
+			'name.errors: 1',
+			'message.id: 1',
+			'message.formId: 1',
+			'message.errorId: 1',
+			'message.descriptionId: 1',
+			'message.initialValue: 1',
+			'message.value: 1',
+			'message.key: 1',
+			'message.dirty: 1',
+			'message.valid: 1',
+			'message.errors: 1',
```

Comment:
@edmundhung I'm not sure if this is concerning to you or not, but the values are different between OS's if you have an idea on how to make this more deterministic I'm open to hearing your feedback.

Reply:
This seems to be fixed? I guess you might had issue with the strict mode setup.

Reply:
@edmundhung actually running locally on my machine, the results differ from CI, not sure why

Reply:
Those number are the render count and will be different dependings whether you are running react in dev with strict mode (e.g. double renders) or in production. To get a stable number, I did some [simple math](https://github.com/edmundhung/conform/blob/be32a14669dc62fd8a470b062c80a18b027068e7/playground/app/routes/subscription.tsx#L136-L148) here to normalize the value based on whether the strict mode is running. This is decided by the CI env [here](https://github.com/edmundhung/conform/blob/be32a14669dc62fd8a470b062c80a18b027068e7/playground/app/routes/subscription.tsx#L40). 

I usually run the playground in dev mode locally so the env doesn't matter. But if you run the test without a running playground, it will run the playground in production mode without the CI env. That's why the number wrong.